vast/vast.cpp

```diff
@@ -77,6 +77,11 @@ int main(int argc, char** argv) {
   // From here on, options from the command line can be used.
   detail::merge_settings(invocation->options, cfg.content,
                          policy::merge_lists::yes);
+  // Tweak CAF parameters in case we're running a client command.
+  bool is_server = invocation->full_name == "start"
+                   || caf::get_or(cfg.content, "vast.node", false);
+  if (!is_server)
+    cfg.set("scheduler.max-threads", 1);
```

Comment:
Can you add a comment here to adjust this with a `caf.` prefix when updating to CAF 0.18+?